scripts/mongo/createTables.php

```diff
@@ -82,10 +82,10 @@ function showUsage()
  */
 function generateTables($id, $tableId, $storeName, $stat = null)
 {
-    $tableSpec = MongoTripodConfig::getInstance()->getTableSpecification($storeName, $tableId);
+    $tableSpec = TripodConfig::getInstance()->getTableSpecification($storeName, $tableId);
```

Comment:
We seem to have \Tripod\Mongo\Config in other places, but TripodConfig here. Is this correctly named?